HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -25,6 +25,15 @@ def modifyHLTPhaseIPixelGeom(process):
 
 # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
 def customizeHLTForPFTrackingPhaseI2017(process):
+    # Bit of a hack to replace a module/sequence with another without
+    # expanding the sequences containing it, but works
+    #
+    # In principle setattr(process) can be used to replace a
+    # module/sequence too, but it expands the sequences making life
+    # more complex
+    modifier = cms.Modifier()
```

Comment:
HI @makortel - i really don't understand the role of modifier here. Its not the way they were designed to be used (but thats not to say we shouldn't use them this way - rather its hard to naively understand what happens..)